packages/chat-client/src/constants/chatInvites.ts

```diff
@@ -1,2 +1,4 @@
-export const CHAT_SENT_INVITES_CONTEXT = "chat_sent_invites";
+export const CHAT_SENT_INVITES_CONTEXT = "com.walletconnect.chat.sentInvites";
+export const CHAT_RECEIVED_INVITES_STATUS_CONTEXT =
+  "com.walletconnect.chat.receivedInviteStatuses";
 export const CHAT_RECEIVED_INVITES_CONTEXT = "chat_received_invites";
```

Comment:
Does this need to change too?

Reply:
What is "this" referring to, which store?

(very javascripty question :joy: )

Reply:
Lol, I meant that the formatting change towards namespacing seems to have been applied to all the constants except `CHAT_RECEIVED_INVITES_CONTEXT`, so seemed like the odd one out